### PR TITLE
Add Front-End design elements

### PR DESCRIPTION
**Pull request by gauthamzz**
**https://github.com/gauthamzz/FrontendDesignElements**

**[Its a list of popular quick look up front end design elements.]**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by davisonio** [comment]
Thanks for submitting but it does not meet the requirements above. This list will require a lot of work.

Please leave a comment here once the requirements have been fulfilled and we can reopen this.

**Comment by davisonio** [comment]
Largely, the items could do with some descriptions explaining why they are awesome etc. check other lists for how it is done e.g. https://github.com/nicolesaidy/awesome-web-design

**Comment by gauthamzz** [comment]
It has been updated and descriptions are added. @davisonio

**Comment by gauthamzz** [comment]
I hope the requirements are fulfilled now. Please take a look.

All of these are met 
------------------------
I have read and understood the contribution guidelines and the instructions for creating a list.
This pull request has a descriptive title. For example, Add Name of List, not Update readme.md or Add awesome list.
The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
The list I added
has been around for at least 30 days,
is the result of hard work and is the best I could possibly produce,
is a non-generated Markdown file in a GitHub repo,
is not a duplicate,
includes a succinct description of the project/theme at the top (example),
includes a project logo/illustration (whenever possible) at the top right of the readme (example). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
only has awesome items with descriptions (Awesome lists are curations of the best, not of everything),
contains the awesome badge on the right side of the list heading,
has a Table of Contents section named Contents as the first section,
has an appropriate license (Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc) (If you use a license badge, it should be SVG, not PNG),
has contribution guidelines,
doesn't include a Travis badge,
has been checked for inconsistent formatting, spelling, and grammar:
Each link description starts with an uppercase character and ends with a period.
Drop all the A / An prefixes in the descriptions.
Consistent naming. For example, Node.js, not NodeJS or node.js.

**Comment by rroderickk** [comment]
Currated, Friend I have a question for you urgently, do you recommend
investing my savings in bitcoin? You are wise and you take more time than
me in the world of programming mentor guide me.


*RODRIGO ANDRES MORENO PERTUZ*
*Ingeniero en Minas*, Especialista en Geotecnia Vial.
*UNIVERSIDAD PEDAGÓGICA Y TECNOLÓGICA DE COLOMBIA**.*
*Contacto.: 301 55 888 00  - r
<ing_dga@hotmail.com>odrigo.moreno@uptc.edu.co <odrigo.moreno@uptc.edu.co>*

2016-12-19 15:34 GMT-05:00 Gautham Santhosh <notifications@github.com>:

**Comment by gauthamzz** [comment]
merge conflict solved , review @sindresorhus @davisonio

File: readme.md [hunk 1]
```diff
@@ -189,7 +189,8 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Redux](https://github.com/brillout/awesome-redux) - State container for JavaScript apps.
 - [webpack](https://github.com/d3viant0ne/awesome-webpack) - Module bundler.
 - [Browserify](https://github.com/ungoldman/awesome-browserify) - Module bundler.
-- [Sass](https://github.com/Famolus/awesome-sass) - CSS preprocessor.
+- [Sass](https://github.com/Famolus/awesome-sass) - CSS preprocess
+- [Frontend Design Elements](https://github.com/gauthamzz/FrontendDesignElements) - Front-End design elements .
```
Comment (davisonio) [inline readme.md]: Got a trailing full stop here and you cut off the `or.` on the point above

Comment (gauthamzz) [inline readme.md]: Sorry , accidental, @davisonio fixed now

**Comment by gauthamzz** [comment]
Is it okay now @davisonio

**Comment by davisonio** [comment]
> Is it okay now @davisonio

missing the or from processor.

**Comment by gauthamzz** [comment]
@davisonio any other problems

**Comment by sindresorhus** [comment]
I'm gonna pass on this for now. Right now I don't see enough value in this list. For me it's just a random list of Codepen examples, while a few are cool, I don't think I would actually use any of them in production. The list itself also needs a lot of work and many things from the guidelines are not followed.

I would recommend adding it to https://github.com/sindresorhus/awesome/wiki instead.

**Pull request closed without merging**